### PR TITLE
Add WhoCanShare AllowLists to SetSPOTenant.md

### PR DESCRIPTION
**Pull request by hhhults**

### CODE REVIEW

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md [hunk 1]
```diff
@@ -3526,6 +3528,60 @@ Accept pipeline input: False
...
+To set this list to be a specific security group, you need to enter its GUID as the parameter. You can enter multiple GUIDs by using a comma to separate them, for example Set-SPOTenant –WhoCanShareAnonymousAllowList 46698648-fcd5-41fc-9526-c7f7b2ace919,544dd15b-cf3c-441b-96da-004d5a8cea1d. To view the current list, use Get-SPOTenant to get WhoCanShareAnonymousAllowList. 
+
+To add a security group to this list, use 
+
+```powershell
+$list = (Get-SPOTenant | Select-Object WhoCanShareAnonymousAllowList).WhoCanShareAnonymousAllowList 
+Set-SPOTenant –WhoCanShareAnonymousAllowList ($list + <new guid>) 
+``` 
+
+To remove all security groups from this list, use Set-SPOTenant –WhoCanShareAnonymousAllowList @().
```
Comment (samkabue) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:3544]: I'd suggest moving these to EXAMPLES section, and leaving only the param description here.

i.e. add ~# EXAMPLE 19` etc -- however many examples you need

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md [hunk 2]
```diff
@@ -3526,6 +3528,60 @@ Accept pipeline input: False
...
+To set this list to be a specific security group, you need to enter its GUID as the parameter. You can enter multiple GUIDs by using a comma to separate them, for example Set-SPOTenant –WhoCanShareAuthenticatedGuestAllowList 46698648-fcd5-41fc-9526-c7f7b2ace919,544dd15b-cf3c-441b-96da-004d5a8cea1d. To view the current list, use Get-SPOTenant to get WhoCanShareAuthenticatedGuestAllowList. 
+
+To add a security group to this list, use 
+
+```powershell
+$list = (Get-SPOTenant | Select-Object WhoCanShareAuthenticatedGuestAllowList). WhoCanShareAuthenticatedGuestAllowList 
+Set-SPOTenant – WhoCanShareAuthenticatedGuestAllowList ($list + <new guid>) 
+``` 
+
+To remove all security groups from this list, use Set-SPOTenant – WhoCanShareAuthenticatedGuestAllowList @().
```
Comment (samkabue) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:3571]: same re: moving these to EXAMPLES

**Comment by hhhults** [comment]
@microsoft-github-policy-service agree [company="Microsoft"]

**Comment by hhhults** [comment]
@microsoft-github-policy-service agree company="Microsoft"

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md [hunk 3]
```diff
@@ -3526,6 +3544,42 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -WhoCanShareAnonymousAllowList 
+
+Sets the list of security groups who are allowed to share with anonymous (non-authenticated) users as well as authenticated guest users. Each security group is denoted by its GUID object ID in the Entra directory. 
+
+To set this list to be a specific security group, you need to enter its GUID as the argument. You can enter multiple GUIDs by using commas to separate them. To view the current list, use Get-SPOTenant.
```
Comment (samkabue) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:3551]: >Get-SPOTenant [](http://example.com/codeflow?start=187&length=13)

link this to the Get-SPOTenant documentation. Use relative link

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md [hunk 4]
```diff
@@ -3526,6 +3544,42 @@ Accept pipeline input: False
...
+Default value: None 
+Accept pipeline input: False 
+Accept wildcard characters: False 
+``` 
+
+### -WhoCanShareAuthenticatedGuestAllowList 
+
+Sets the list of security groups who are only allowed to share with authenticated guest users. Each security group is denoted by its GUID object ID. 
+
+To set this list to be a specific security group, you need to enter its GUID as the argument. You can enter multiple GUIDs by using commas to separate them. To view the current list, use Get-SPOTenant.
```
Comment (samkabue) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:3569]: >Get-SPOTenant [](http://example.com/codeflow?start=187&length=13)

link this to the Get-SPOTenant documentation. Use relative link

**Comment by samkabue** [comment]
> [Upgrade-SPOSite](Upgrade-SPOSite.md)

could you help remove this one? We don't really use this cmdlet anymore.

Instead, add the `Get-SPOSite` documentation link here

---
Refers to: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:3587 in e056025. [](commit_id = e056025afc05a940a4fda6e6a7cc3e67d5d13541, deletion_comment = False)

**Review by samkabue — APPROVED** [review]
:shipit:

**Pull request merged**